kubernetes/test-infra/aks-new/main.tf

```diff
@@ -0,0 +1,65 @@
+provider "azurerm" {
+  features {}
+}
+
+variable "location" {
+  type = string
+  default = "West Europe"
```

Comment:
Attempted to run with `West Europe`, we currently have reached our quota for this region. Changing it to `West US` works just fine.